new_pass_logic.php

```diff
@@ -0,0 +1,62 @@
+<?php include "includes/header.php" ?>
+<?php include "includes/nav.php"; ?>
+<?php
+
+if (isset($_POST['new_password'])) {
```

Comment:
This form can be submitted with a valid token but mismatching passwords and I get the following output:
- Passwords did not match
- Password reset, you can [login](http://localhost:8081/login.php) now